tests/plugins/tableselection/tablecellfocus.js

```diff
@@ -0,0 +1,286 @@
+/* bender-tags: tableselection */
```

Comment:
Tests in this file seem to pass also on the `master`. Could you check it?

Reply:
Fixed.